README.md

```diff
@@ -660,6 +863,27 @@ section above:
 [root@localhost ~]# reboot
 ```
 
+__For Dual NUMA node system__
```

Comment:
There is no difference from the VM side for a dual or single node system the isolcpus are the same.